packages/store/gas-report.json

```diff
@@ -603,73 +603,73 @@
     "file": "test/StoreCoreDynamic.t.sol",
     "test": "testGetDynamicFieldSlice",
     "name": "get field slice (cold, 1 slot)",
-    "gasUsed": 10240
+    "gasUsed": 5609
```

Comment:
these gas changes come from not loading the `_tableId` and `_keyTuple` from storage in the gas test

Reply:
same for all tests or just this one? I noticed some other gas improvements below

Reply:
`wrapping`/`unwrapping` in itself should not significantly impact gas, so i think most of the gas variation in here comes from the changed gas tests